Makefile

```diff
@@ -9,6 +9,13 @@ build:
 tools:
 	GO_BUILD_FLAGS="${GO_BUILD_FLAGS} -v" ./scripts/build_tools.sh
 
+TEMP_TEST_ANALYZER_DIR=/tmp/etcd-test-analyzer
+bin/etcd-test-analyzer: $(TEMP_TEST_ANALYZER_DIR)/*
+	GO_BUILD_FLAGS="${GO_BUILD_FLAGS} -v" ./scripts/build_etcd_test_analyzer.sh
```

Comment:
Not super convinced that we should put it as a separate script. Building an it seems simple enough that it can be put inline in makefile.